solr/contrib/analytics/src/java/org/apache/solr/analytics/facet/RangeFacet.java

```diff
@@ -54,7 +53,7 @@ public RangeFacet(String name, SchemaField field, String start, String end, List
   }
 
   @Override
-  public void createFacetValueExecuters(final Filter filter, SolrQueryRequest queryRequest, Consumer<FacetValueQueryExecuter> consumer) {
+  public void createFacetValueExecuters(final Query query, SolrQueryRequest queryRequest, Consumer<FacetValueQueryExecuter> consumer) {
```

Comment:
"filter" is a reasonable name

Reply:
TODO still

Reply:
Thanks, missed it in the initial revision. Fixed it now!